src/burn/engine/plan.cpp

```diff
@@ -1922,7 +1923,8 @@ static HRESULT GetActionDefaultRequestState(
     {
     case BOOTSTRAPPER_ACTION_INSTALL: __fallthrough;
     case BOOTSTRAPPER_ACTION_UPDATE_REPLACE: __fallthrough;
-    case BOOTSTRAPPER_ACTION_UPDATE_REPLACE_EMBEDDED:
+    case BOOTSTRAPPER_ACTION_UPDATE_REPLACE_EMBEDDED: __fallthrough;
+    case BOOTSTRAPPER_ACTION_CACHE:
```

Comment:
I think _CACHE should go before _INSTALL here since that matches the order of the enum better.